modules/library.js

```diff
@@ -0,0 +1,125 @@
+         removeButton.id = (Library.book.length - 1);
+         div.appendChild(removeButton);
+         const removeBtn = document.querySelectorAll('.removee');
+         removeBtn.forEach((item) => item.addEventListener('click', Library.removeBook));
+         Library.clearFields();
+         Library.showSuccessMessage();
+       }
+     }
+
+     static checkBookInData() {
+       bookExist.style.display = ('block');
+       Library.clearFields();
+       Library.removeMessageOnInput();
+     }
+
+     static clearFields() {
+       bookTitle.value = ('');
+       bookAuthor.value = ('');
+     }
+
+     static showSuccessMessage() {
+       const bookAdded = document.querySelector('.bookAdded');
+       bookAdded.style.display = ('block');
+       Library.removeMessageOnInput();
+     }
+
+     static removeMessageOnInput() {
+       const bookAdded = document.querySelector('.bookAdded');
+       input.forEach((item) => {
+         item.addEventListener('focus', () => {
+           bookAdded.style.display = ('none');
+           bookExist.style.display = ('none');
+         });
+       });
+     }
+
+     static removeBook(ev) {
+       const elementToRemove = document.querySelectorAll('[data-id]');
+       elementToRemove.forEach((item) => {
+         if (item.dataset.id === ev.target.id) {
+           item.style.display = ('none');
+         }
+       });
+       for (let i = 0; i < Library.book.length; i += 1) {
+         if (i.toString() === ev.target.id) {
+           Library.book.splice(i, 1);
+           localStorage.setItem('list', JSON.stringify(Library.book));
+         }
+       }
+     }
+
+     static init() {
+       for (let i = 0; i < Library.book.length; i += 1) {
+         const section = document.querySelector('.bookSection');
+         const div = document.createElement('div');
+         div.className = ('bookList');
+         div.dataset.id = (i);
+         const bookInfo = document.createElement('p');
+         bookInfo.textContent = (`"${Library.book[i].title}" by ${Library.book[i].author}`);
+
+         console.log(data);
+
+         bookInfo.className = ('bookInfo');
+         div.appendChild(bookInfo);
+         const removeButton = document.createElement('button');
+         removeButton.textContent = ('Remove');
+         removeButton.className = ('removee');
+         removeButton.id = (i);
+         div.appendChild(removeButton);
+         section.appendChild(div);
+         const removeBtn = document.querySelectorAll('.removee');
+         removeBtn.forEach((item) => item.addEventListener('click', Library.removeBook));
+       }
+     }
+
+     static onloadFunction() {
+       const empty = document.querySelector('.emptyCollection');
+       const data = JSON.parse(localStorage.getItem('list'));
+       console.log(data);
+       if (data[0] !== undefined) {
+         empty.style.display = ('none');
+         Library.book = data;
+
+         console.log(Library.book);
```

Comment:
- [ ] Please do not commit `console.log` to your repo. As it’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. Please it will be nice to remove them.

Reply:
:100: We removed it :smile: